ui/lib/kubernetes/addon/components/page/roles.hbs

```diff
@@ -43,9 +43,7 @@
         <Item.menu as |Menu|>
           {{#if role.rolesPath.isLoading}}
             <li class="action">
-              <button disabled type="button" class="link button is-loading is-transparent">
-                loading
-              </button>
+              <Hds::Button disabled @color="tertiary" @icon="loading" @text="loading" @isIconOnly={{true}} />
```

Comment:
Since this button has the `is-transparent` class it was skipped by the codemod (a rule that we wrote in deliberately). There were only 3 cases I had to update manually, all exactly like this. If this case is prevalent we can look at writing a transform for it in the codemod.